arangod/Pregel/PregelFeature.cpp

```diff
@@ -361,8 +362,11 @@ ResultT<ExecutionNumber> PregelFeature::startExecution(TRI_vocbase_t& vocbase,
 
   _actorRuntime->spawn<conductor::ConductorActor>(
       vocbase.name(),
-      std::make_unique<conductor::ConductorState>(executionSpecifications,
-                                                  vocbase),
+         auto vocbaseLookupInfo = std::make_unique<conductor::VocbaseLookupInfo>(
```

Comment:
As Markus already mentioned: This will not work, you have to put the auto vocbaseLookupInfo definition before line 363.

Reply:
Just a side-note: The lookup is now done here instead of inside the CreateWorkers state, but I actually like that. It fits the other stuff done in this PregelFeature-function `startExecution`.